### DISCUSSION TITLE
Design component file

### DISCUSSION THREAD

**Discussion by mariaclaratgolfo** [vmware/clarity#6094] [Q&A]
I notice on the website you only provide a design file in [Figma](https://clarity.design/get-started/design/) to leverage the component library. While I adore Figma, my client is using XD and Sketch. Is there any chance a file can be provided for use in these tools? I specifically hope to leverage your Angular components.

**Comment by bbogdanov** [comment]
Hi @mariaclaratgolfo,

Did you try this one https://www.figma.com/community/plugin/849159306117999028/Convertify-Figma-to-Sketch%2FXD ?

**Reply by mariaclaratgolfo** [reply]
I can try this! Thank you. Hopefully it converts nicely. Often if it's not originally created in the tool the layers get borked.

**Reply by bbogdanov** [reply]
What we use is Figma and unfortunately we are not able to provide angular components that were made originally in XD or Sketch
